src/test/kotlin/week1/calculator/OperatorTypeTest.kt

```diff
@@ -0,0 +1,49 @@
+package week1.calculator
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.EnumSource
+import org.junit.jupiter.params.provider.ValueSource
+
+class OperatorTypeTest {
+
+    @Test
+    fun `더하기`() {
+        Assertions.assertThat(OperatorType.ADD.execute(1, 2)).isEqualTo(3)
+        Assertions.assertThat(OperatorType.ADD.execute(123000, 456)).isEqualTo(123456)
+    }
+
+    @Test
+    fun `빼기`() {
+        Assertions.assertThat(OperatorType.SUBTRACT.execute(1, 2)).isEqualTo(-1)
+        Assertions.assertThat(OperatorType.SUBTRACT.execute(10, 2)).isEqualTo(8)
+    }
+
+    @Test
+    fun `곱하기`() {
+        Assertions.assertThat(OperatorType.MULTIPLY.execute(1, 2)).isEqualTo(2)
+        Assertions.assertThat(OperatorType.MULTIPLY.execute(123, 456)).isEqualTo(56088)
+    }
+
+    @Test
+    fun `나누기`() {
+        Assertions.assertThat(OperatorType.DIVIDE.execute(1, 2)).isEqualTo(0)
+        Assertions.assertThat(OperatorType.DIVIDE.execute(100, 9)).isEqualTo(11)
+        Assertions.assertThat(OperatorType.DIVIDE.execute(100, 0)).isEqualTo(100)
+    }
```

Comment:
`@MethodSource`를 활용하면 하나의 테스트 케이스로 변경해볼 수 있을 것 같아요. 🤗
```suggestion
    @ParameterizedTest
    @MethodSource
    fun `연산을 한다`(sut: OperatorType, a: Int, b: Int, expected: Int) {
        // when
        val actual = sut.execute(a, b)

        // then
        assertThat(actual).isEqualTo(actual)
    }

    companion object {
        @JvmStatic
        fun `연산을 한다`(): List<Arguments> {
            return listOf(
                Arguments.of(OperatorType.ADD, 1, 2, 3),
                Arguments.of(OperatorType.ADD, 123000, 456, 123456),
                Arguments.of(OperatorType.SUBTRACT, 1, 2, -1),
                Arguments.of(OperatorType.MULTIPLY, 1, 2, 2),
                Arguments.of(OperatorType.DIVIDE, 1, 2, 0),
            )
        }
    }
```

Reply:
오.. 훨씬 깔끔하네요!